suites/squid/rgw/tier-2_rgw_module.yaml

```diff
@@ -0,0 +1,94 @@
+# global conf : /conf/squid/tier-0_rgw.yaml
+# Rgw  deployed with rgw module , bootstrap method
+tests:
+
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                registry-url: registry.redhat.io
+                mon-ip: node1
+                initial-dashboard-password: admin@123
+                dashboard-password-noupdate: true
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: bootstrap with registry-url option and deployment services.
+      destroy-cluster: false
+      polarion-id: CEPH-83573713
+      module: test_cephadm.py
+      name: RHCS deploy cluster using cephadm
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node6
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+        git_clone: true
+        git_node_role: rgw
+      desc: Configure the RGW client system
+      polarion-id: CEPH-83573758
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+  - test:
+      abort-on-fail: true
+      config:
+        cephadm: false
+        role: client
+        commands:
+          - "ceph mgr module enable rgw"
+          - |
+            cat <<EOF > /tmp/zone-spec.yaml
+            service_type: rgw
+            service_id: rgw.1
+            service_name: rgw.rgw.1
+            placement:
+              hosts:
+                - {node:node5}
```

Comment:
@yuva-gayam , the conf - /conf/squid/tier-0_rgw.yaml, has 3 rgw daemons added. Could you update the hosts accordingly.